docs/index.rst

```diff
@@ -17,29 +17,29 @@
 Welcome to NVIDIA Dynamo
 ========================
 
-The NVIDIA Dynamo Platform is a high-performance, low-latency inference framework designed to serve all AI models—across any framework, architecture, or deployment scale.
+The NVIDIA Dynamo Platform is a high-performance, low-latency inference framework designed to serve generative AI and reasoning models—across any framework, architecture, or deployment scale. Built in Rust for performance and in Python for extensibility, Dynamo is fully open-source and driven by a transparent, OSS (Open Source Software) first development approach.
+
+Large language models are quickly outgrowing the memory and compute budget of any single GPU. Tensor-parallelism solves the capacity problem by spreading each layer across many GPUs—and sometimes many servers—but it creates a new one: how do you coordinate those shards, route requests, and share KV cache fast enough to feel like one accelerator? This orchestration gap is exactly what NVIDIA Dynamo is built to close.
 
 .. admonition:: 💎 Discover the latest developments!
    :class: seealso
 
    This guide is a snapshot of the `Dynamo GitHub Repository <https://github.com/ai-dynamo/dynamo>`_ at a specific point in time. For the latest information and examples, see:
 
    - `Dynamo README <https://github.com/ai-dynamo/dynamo/blob/main/README.md>`_
-   - `Architecture and features doc <https://github.com/ai-dynamo/dynamo/blob/main/docs/architecture/>`_
-   - `Usage guides <https://github.com/ai-dynamo/dynamo/tree/main/docs/guides>`_
-   - `Dynamo examples repo <https://github.com/ai-dynamo/examples>`_
+   - `Architecture and Features <https://github.com/ai-dynamo/dynamo/blob/main/docs/architecture/>`_
+   - `Usage Guides <https://github.com/ai-dynamo/dynamo/tree/main/docs/guides>`_
+   - `Dynamo Examples <https://github.com/ai-dynamo/dynamo/tree/main/examples>`_
 
 
 Quick Start
 -----------------
 Follow the :doc:`Quick Guide to install Dynamo Platform <guides/dynamo_deploy/quickstart>`.
```

Comment:
Do you think this should be the "dynamo quickstart"? Or should there be something simpler like `pip install dynamo - python3 -m dynamo.frontend - python3 -m dynamo.trtllm` as the front page quickstart?

Maybe we can have two here on front page:

Quickstart
1. Local Deployment (link to easy pip install + commands)
2. Kubernetes Deployment (link to dynamo_deploy quickstart)